app/predacons_cli/src/cli.py

```
@@ -0,0 +1,45 @@
+        
+    
+    def settings(self):
+        return self.predacons.settings()
+    
+    def check_config_file(self):
+        file_path = 'predacon_cli_config.json'
+
+        if os.path.exists(file_path):
+            return True
+        else:
+            return False
```

Ruff

Return the condition `bool(os.path.exists(file_path))` directly

Replace with `return bool(os.path.exists(file_path))`

(SIM103)